packages/stores/src/account/cosmos/index.ts

```diff
@@ -124,6 +124,10 @@ export class CosmosAccountImpl {
       );
     }
 
+    const revisionNumber = ChainIdHelper.parse(
+      destinationInfo.network
+    ).version.toString();
+
     const msg = this.msgOpts.ibcTransfer.messageComposer({
```

Comment:
Did you remove our bespoke IBC transfer logic/files?

Reply:
This is a port from Keplr's `sendIbcMsg` that we used in our previous `AccountStore`. All other IBC transfer files are the same.